src/footer/index.md

```diff
@@ -0,0 +1,48 @@
+---
+template: footer-nav
+email_link:
+  label: Email Us
+  email: work@keep.network
+nav_dropdowns:
+  - title: Company
+    dropdown_items:
```

Comment:
I'll be honest, it feels like it would make more sense to call these “nav categories” and “items” rather than “nav dropdowns” and “dropdown items”. At the CMS level, it feels like we should be talking about content rather than the layout. These aren't always dropdowns, either---that's only the case at narrow widths.

Reply:
Good point! Will rename later.